pkg/errorResponses/errorResponses.go

```diff
@@ -0,0 +1,44 @@
+// SPDX-FileCopyrightText: 2023 Greenbone AG <https://greenbone.net>
+//
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+// Package errorResponses provides rest api models for errors
+package errorResponses
+
+import "fmt"
+
+type ErrorResponse struct {
+	Type    string `json:"type"`
+	Title   string `json:"title"`
+	Details string `json:"details,omitempty"`
+}
+
+type FieldErrorResponse struct {
```

Comment:
@mgoetzegb how about just having a single ErrorResponse struct with ```Errors  map[string]string `json:"errors,omitempty"``` and two "constructors" ( ``` NewErrorGenericResponse``` and ```NewFieldErrorResponse```)?

Reply:
Sounds good to me, having a single type for the error response might simplify thing.

Changed as suggested, but named the constructor `NewErrorValidationResponse` to align it with the name of the error type (`greenbone/validation-error`).